app/questionnaire/navigation.py

```diff
@@ -179,3 +191,11 @@ def _generate_link_names(self, repeating_rule):
             link_names[link_name] = " ".join(link_names[link_name])
 
         return link_names
+
+    @staticmethod
+    def _all_sections_completed(navigation):
```

Comment:
Agreed with Andrew and Chris this will be tackled in the overall navigation improvements. Its what my comment was about. I have added the information on the Trello card so this PR can progress without it, it won't affect anything in flight at present